resources/lib/videoplayer.py

```diff
@@ -41,12 +41,16 @@ def __init__(self):
         self._stream_data: VideoPlayerStreamData | None = None  # @todo: maybe rename prop and class?
         self._player: Optional[xbmc.Player] = xbmc.Player()  # @todo: what about garbage collection?
         self._skip_modal_duration_max = 10
+        self.waitForStart = True
+        self.lastUpdatePlayhead = 0
+        self.cleardStream = False
```

Comment:
Only a minor, but that would be `cleared` :)

Reply:
fixed